### PR TITLE
Changed 'wakeword' to 'audiosample' to better reflect the backend table.

### PR DESCRIPTION
**Pull request by snekiam**
## What's New?
This addresses #44 - Refactor API for data upload.

## Type of change (pick-one)
- [ ] Bug fix (_non-breaking change which fixes an issue_)
- [ ] New feature (_non-breaking change which adds functionality_)
- [x] Breaking change (_fix or feature that would cause existing functionality to not work as expected_)
- [ ] This change requires a documentation update

## How Has This Been Tested?
Tested locally using firefox/postman

## Checklist (check-all-before-merge)
_formatting help: `- [x]` means "checked' and `- [ ]` means "unchecked"_

- [ ] I documented my code according to the [Google Python Style Guide][1]

- [ ] I ran `./build_docs.sh` and the docs look fine

- [ ] I ran `./type_check.sh` and got no errors

- [ ] I ran `./format.sh` because it automatically cleans my code for me 😄 

- [ ] I ran `./lint.sh` to check for what "format" missed

- [ ] I added my tests to the `/tests` directory

- [ ] I ran `./run_tests.sh` and all the tests pass

[1]: https://google.github.io/styleguide/pyguide.html

### CODE REVIEW

**Comment by snekiam** [comment]
This *will* break the current web recorder until it is updated to reflect the new endpoint.

**Comment by mfekadu** [comment]
@snekiam 

I was thinking it could be as simple as `@app.route('/audiosample', methods=['POST'])` such that the fact that it is a POST request implies new data is being added

File: flask_api.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def handle_question():
     return jsonify(response), SUCCESS
 
 
-@app.route('/new_data/wakeword', methods=['POST'])
+@app.route('/new_data/audiosample', methods=['POST'])
```
Comment (mfekadu) [inline flask_api.py:72]: let's go with `/audiosample` to keep it simple

**Review by mfekadu — CHANGES_REQUESTED** [review]

File: flask_api.py (hunk 1 above)
Comment (mfekadu) [reply flask_api.py:72]: @Waidhoferj what are your thoughts?

Comment (snekiam) [reply flask_api.py:72]: I have no preference either way.

**Comment by snekiam** [comment]
Closing since this is super small and super old.

**Pull request closed without merging**